docs/world api.md

```diff
@@ -696,9 +696,73 @@ When importing a file that defines a class that inherits from `worlds.AutoWorld.
 is automatically extended by the mixin's members. These members should be prefixed with the name of the implementing
 world since the namespace is shared with all other logic mixins.
 
-Some uses could be to add additional variables to the state object, or to have a custom state machine that gets modified
-with the state.
-Please do this with caution and only when necessary.
+In general, a LogicMixin class should have at least one mutable variable that is tracking some custom state per player,
+as well as an `init_mixin` and a `copy_mixin` function so that this variable gets initialized and copied correctly when
```

Comment:
These are different noun types, so you'd want to make them the same, this way I think is best:
```suggestion
as well `init_mixin` and `copy_mixin` functions so that this variable gets initialized and copied correctly when
```

Reply:
Uhhh that should still say "as well as" right?

Reply:
```suggestion
as well as `init_mixin` and `copy_mixin` functions so that this variable gets initialized and copied correctly when
```

Reply:
🤦